pages/docs/tutorials/generate-code.md

```diff
@@ -6,12 +6,13 @@ weight: 100
 
 ## Introduction
 
-In this tutorial, you'll learn how to generate code from your AsyncAPI document using the AsyncAPI generator tool.
+In this tutorial, you'll learn how to generate code from your AsyncAPI document using [Glee](https://github.com/asyncapi/glee).
 
 ## Background context
-The [AsyncAPI Generator](https://github.com/asyncapi/generator) is a tool that you can use to generate whatever you want based on the AsyncAPI document. You can generate docs and code. It can be used as a library in a Node.js application or through the [AsyncAPI CLI](https://github.com/asyncapi/cli).
+[Glee](https://github.com/asyncapi/glee) is a TS/JS framework that enables you to create APIs and messaging clients based on the AsyncAPI document. Instead of generating code, this framework tightly integrates with your AsyncAPI document and binds functions to specific
+AsyncAPI operations. You only have to provide the code for these functions and Glee will take care of the rest.
 
-The generator tool supports a number of templates to generate code for a variety of different languages and protocols as the output. These templates help to specify what exactly must be generated, and in this tutorial, you'll use a [Node.js template](https://github.com/asyncapi/nodejs-template).
+Glee is often used with the [AsyncAPI CLI](/tools/cli) for a better development experience.
```

Comment:
```suggestion
Glee is often used with the [AsyncAPI CLI](/tools/cli) for a better development experience.

In the previous tutorial, you created an AsyncAPI document that is used in this tutorial.

<Remember>

If you did not follow the previous tutorial and do not have an `asyncapi.yaml` file for overview, then generate one by running the following command using AsyncAPI CLI: 
`asyncapi new --example=tutorial.yml --no-tty`.

</Remember>
```


☝🏼 will work once we merge https://github.com/asyncapi/cli/pull/888 before release